source/manual/architecture-shallow-dive.html.md

```diff
@@ -0,0 +1,204 @@
+
+Within Origin, the request goes to an application called Router to determine which GOV.UK application will produce the page.
+
+The request from Router is then proxied (or forwarded) to an application called Government Frontend which produces the homepage as an HTML response and sends that response back to Router. Router sends the response to Fastly. Fastly sends the response back to the user.
+
+### A user visits a GOV.UK webpage
+
+The user searches for a GOV.UK webpage, and selects the webpage link in the search results. For example, this [news page about UK changes to travel rules for travellers from Nigeria to England](https://www.gov.uk/government/news/uk-changes-travel-rules-for-travellers-from-nigeria-to-england).
+
+The user's web browser checks if it has a locally cached version of the webpage by sending a request to the Fastly content delivery network (CDN). A locally cached version is more likely if the user has visited the webpage in the last 30 mins.
+
+If Fastly has the webpage cached, Fastly returns the webpage to the user.
+
+If Fastly does not have the webpage cached, Fastly sends the request on to Origin. Origin is the infrastructure that GOV.UK maintains, hosted in Amazon Web Services (AWS).
+
+Within Origin, the request goes to an application called Router to determine which GOV.UK application will render the page.
+
+Router gets the request and decides which application should render the page. Router then proxies (forwards) the request to the Government Frontend application.
+
+Government Frontend looks at request and decides which content to get from the Content Store. Government Frontend then queries the Content Store API to get the content in an [API response from the Content Store](https://www.gov.uk/api/content/government/news/uk-changes-travel-rules-for-travellers-from-nigeria-to-england).
+
+Government Frontend then builds the page using the content in the API response, and sends that response back to Router.
+
+Router sends the response to Fastly. Fastly sends the response back to the user.
+
+### A user searches GOV.UK using the search page
+
+The user goes to the GOV.UK search page, and searches for something.
+
+The user's web browser checks if it has a locally cached version of the search results by sending a request to the Fastly content delivery network (CDN). A locally cached version is more likely if the user has made the same search in the last 5 mins.
+
+If Fastly has the search results page cached, Fastly returns the results page to the user.
+
+If Fastly does not have the search results page cached, Fastly sends the request on to Origin. Origin is the infrastructure that GOV.UK maintains, hosted in Amazon Web Services (AWS).
+
+Within Origin, the request goes to an application called Router. The Finder Frontend application serves search results pages, so Router sends the request to Finder Frontend.
```

Comment:
Again, I think we can avoid repeating ourselves - and also the distinction between a 30m cache and a 5m cache is too low-level detail for an introduction to architecture doc.

```suggestion
As above, the request goes through Fastly, to Origin, to the Router application. This time, Router knows that the route should be handled by the Finder Frontend application.
```

Reply:
As per previously stated reasons, would prefer to keep detail in this section in case read out of sequence.